rsmtool/rsmtool.py

```diff
@@ -97,13 +97,27 @@ def run_experiment(config_file_or_obj,
     writer = DataWriter(configuration['experiment_id'])
 
     # Get the paths and names for the DataReader
+
     (file_names,
-     file_paths) = configuration.get_names_and_paths(['train_file', 'test_file',
-                                                      'features', 'feature_subset_file'],
-                                                     ['train', 'test', 'feature_specs',
-                                                      'feature_subset_specs'])
+     file_paths_org) = configuration.get_names_and_paths(['train_file', 'test_file',
+                                                          'features', 'feature_subset_file'],
+                                                         ['train', 'test', 'feature_specs',
+                                                          'feature_subset_specs'])
+
+    file_paths = DataReader.locate_files(file_paths_org, configpath)
+
+    # check that we were able to locate all files
+    if None in file_paths:
+        indices_with_no_paths = [i for i in range(len(file_paths))
+                                 if file_paths[i] is None]
+
+        error_message_list = ["{} file {} not found.".format(file_names[i],
+                                                             file_paths_org[i])
+                              for i in indices_with_no_paths]
+
+        raise FileNotFoundError('\n'.join(error_message_list))
+
```

Comment:
Do we need this check here if we're going to get a similar `ValueError` when we initialize the `DataReader` in line 129? Is this just pre-empting that?

Reply:
This will give a more specific error message saying what file is missing at what path. Unfortunately, we don't pass the original path to the DataReader to return a meaningful message and I find them very useful.

Reply:
I am not a fan of the way this exception looks for multiple files. Why not do something like:

```python
    if None in file_paths:
        indices_with_no_paths = [i for i in range(len(file_paths))
                                 if file_paths[i] is None]
        missing_file_paths = [file_paths_org[i] for i in indices_with_no_paths]
        raise FileNotFoundError('The following files were not found: {}'.format(repr(missing_file_paths)))
```

Reply:
This change would obscure what was the destination of the wrong path (e.g. 'train' or 'test' or 'feature'). I suppose we can try dropping it and see if it gets annoying